recipes/llvm-openmp/all/conanfile.py

```diff
@@ -123,37 +132,112 @@ def generate(self):
         tc = CMakeToolchain(self)
         tc.variables["OPENMP_STANDALONE_BUILD"] = True
         tc.variables["LIBOMP_ENABLE_SHARED"] = self.options.shared
-        tc.variables["OPENMP_ENABLE_LIBOMPTARGET"] = self.options.build_libomptarget
-        # Do not buidl OpenMP Tools Interface (OMPT)
+        tc.variables["OPENMP_ENABLE_LIBOMPTARGET"] = self.options.get_safe("build_libomptarget", False)
+        # Do not build OpenMP Tools Interface (OMPT)
         tc.variables["LIBOMP_OMPT_SUPPORT"] = False
+        # Should not be needed and causes the library to be copied on Windows due to lack of symlink support
+        tc.variables["LIBOMP_INSTALL_ALIASES"] = False
+        # The library file incorrectly includes a "lib" prefix on Windows otherwise
+        if self.settings.os == "Windows":
+            tc.variables["LIBOMP_LIB_NAME"] = "omp"
         tc.generate()
 
     def _patch_sources(self):
         apply_conandata_patches(self)
+        if self._version_major < 17:
+            # Fix CMake version and policies not being propagated in linker tests
+            replace_in_file(self, os.path.join(self.source_folder, "runtime", "cmake", "LibompCheckLinkerFlag.cmake"),
+                            "cmake_minimum_required(",
+                            "cmake_minimum_required(VERSION 3.15) #")
+            # Ensure sufficient CMake policy version is used for tc.variables
+            replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                            "cmake_minimum_required(",
+                            "cmake_minimum_required(VERSION 3.15) #")
+        # Disable tests
         replace_in_file(self, os.path.join(self.source_folder, "runtime", "CMakeLists.txt"),
                         "add_subdirectory(test)", "")
+        # v12 can be built without LLVM includes
         if self._version_major == 12:
-            # v12 can be built without LLVM includes
             replace_in_file(self, os.path.join(self.source_folder, "libomptarget", "CMakeLists.txt"),
                             "if (NOT LIBOMPTARGET_LLVM_INCLUDE_DIRS)", "if (FALSE)")
+        # TODO: looks like a bug, should ask upstream
+        # The built import lib is named "libomp.dll.lib" otherwise, which also causes install() to fail
```

Comment:
Did you ever find an answer upstream?

Reply:
I opened an issue upstream: https://github.com/llvm/llvm-project/issues/88876
Did not find any existing related issues or PRs.